src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaCollection/stores/CollectionInfoStore.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+import {action, autorun, computed, observable} from 'mobx';
+import {ResourceRequester} from 'sulu-admin-bundle/services';
+import type {BreadcrumbItem, BreadcrumbItems, Collection} from '../types';
+
+const COLLECTIONS_RESOURCE_KEY = 'collections';
+
+export default class CollectionInfoStore {
```

Comment:
I think we've talked about renaming this one to `CollectionStore` and move it to the `MediaBundle/Resources/js/stores/CollectionStore/CollectionStore` (so a separate folder as in the other global stores in the AdminBundle).